
Traceback (most recent call last):
  File "/users/GR/mb/jquilez/utils/io_metadata.py", line 101, in <module>
    tab.upsert(new_data, ['SAMPLE_ID'])
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/dataset/persistence/table.py", line 172, in upsert
    row_count = self.update(row, keys, ensure=ensure, types=types)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/dataset/persistence/table.py", line 145, in update
    rp = self.database.executable.execute(stmt)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1991, in execute
    return connection.execute(statement, *multiparams, **params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 914, in execute
    return meth(self, multiparams, params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/sql/elements.py", line 323, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1010, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1146, in _execute_context
    context)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1341, in _handle_dbapi_exception
    exc_info
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/util/compat.py", line 199, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1139, in _execute_context
    context)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/default.py", line 450, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) database is locked [SQL: u'UPDATE chipseq SET "MEMORY"=? WHERE chipseq."SAMPLE_ID" = ?'] [parameters: ('60G', 'gv_020_01_01_chipseq')]
Traceback (most recent call last):
  File "/users/GR/mb/jquilez/utils/io_metadata.py", line 101, in <module>
    tab.upsert(new_data, ['SAMPLE_ID'])
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/dataset/persistence/table.py", line 172, in upsert
    row_count = self.update(row, keys, ensure=ensure, types=types)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/dataset/persistence/table.py", line 145, in update
    rp = self.database.executable.execute(stmt)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1991, in execute
    return connection.execute(statement, *multiparams, **params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 914, in execute
    return meth(self, multiparams, params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/sql/elements.py", line 323, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1010, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1146, in _execute_context
    context)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1341, in _handle_dbapi_exception
    exc_info
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/util/compat.py", line 199, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1139, in _execute_context
    context)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/default.py", line 450, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) database is locked [SQL: u'UPDATE chipseq SET "MAX_TIME"=? WHERE chipseq."SAMPLE_ID" = ?'] [parameters: ('20:00:00', 'gv_020_01_01_chipseq')]
Traceback (most recent call last):
  File "/users/GR/mb/jquilez/utils/io_metadata.py", line 101, in <module>
    tab.upsert(new_data, ['SAMPLE_ID'])
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/dataset/persistence/table.py", line 172, in upsert
    row_count = self.update(row, keys, ensure=ensure, types=types)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/dataset/persistence/table.py", line 145, in update
    rp = self.database.executable.execute(stmt)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1991, in execute
    return connection.execute(statement, *multiparams, **params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 914, in execute
    return meth(self, multiparams, params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/sql/elements.py", line 323, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1010, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1146, in _execute_context
    context)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1341, in _handle_dbapi_exception
    exc_info
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/util/compat.py", line 199, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1139, in _execute_context
    context)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/default.py", line 450, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) database is locked [SQL: u'UPDATE chipseq SET "SLOTS"=? WHERE chipseq."SAMPLE_ID" = ?'] [parameters: ('8', 'gv_020_01_01_chipseq')]
Traceback (most recent call last):
  File "/users/GR/mb/jquilez/utils/io_metadata.py", line 101, in <module>
    tab.upsert(new_data, ['SAMPLE_ID'])
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/dataset/persistence/table.py", line 172, in upsert
    row_count = self.update(row, keys, ensure=ensure, types=types)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/dataset/persistence/table.py", line 145, in update
    rp = self.database.executable.execute(stmt)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1991, in execute
    return connection.execute(statement, *multiparams, **params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 914, in execute
    return meth(self, multiparams, params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/sql/elements.py", line 323, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1010, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1146, in _execute_context
    context)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1341, in _handle_dbapi_exception
    exc_info
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/util/compat.py", line 199, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1139, in _execute_context
    context)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/default.py", line 450, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) database is locked [SQL: u'UPDATE chipseq SET "ASSEMBLY_VERSION"=? WHERE chipseq."SAMPLE_ID" = ?'] [parameters: ('hg38_mmtv', 'gv_020_01_01_chipseq')]
Traceback (most recent call last):
  File "/users/GR/mb/jquilez/utils/io_metadata.py", line 101, in <module>
    tab.upsert(new_data, ['SAMPLE_ID'])
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/dataset/persistence/table.py", line 172, in upsert
    row_count = self.update(row, keys, ensure=ensure, types=types)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/dataset/persistence/table.py", line 145, in update
    rp = self.database.executable.execute(stmt)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1991, in execute
    return connection.execute(statement, *multiparams, **params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 914, in execute
    return meth(self, multiparams, params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/sql/elements.py", line 323, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1010, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1146, in _execute_context
    context)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1341, in _handle_dbapi_exception
    exc_info
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/util/compat.py", line 199, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1139, in _execute_context
    context)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/default.py", line 450, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) database is locked [SQL: u'UPDATE chipseq SET "JOB_NAME"=? WHERE chipseq."SAMPLE_ID" = ?'] [parameters: ('chipseq-16.04', 'gv_020_01_01_chipseq')]
Traceback (most recent call last):
  File "/users/GR/mb/jquilez/utils/io_metadata.py", line 101, in <module>
    tab.upsert(new_data, ['SAMPLE_ID'])
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/dataset/persistence/table.py", line 172, in upsert
    row_count = self.update(row, keys, ensure=ensure, types=types)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/dataset/persistence/table.py", line 145, in update
    rp = self.database.executable.execute(stmt)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1991, in execute
    return connection.execute(statement, *multiparams, **params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 914, in execute
    return meth(self, multiparams, params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/sql/elements.py", line 323, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1010, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1146, in _execute_context
    context)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1341, in _handle_dbapi_exception
    exc_info
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/util/compat.py", line 199, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1139, in _execute_context
    context)
  File "/nfs/software/mb/el6.3/python/envs/.virtualenvs/cpython279/lib/python2.7/site-packages/sqlalchemy/engine/default.py", line 450, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) database is locked [SQL: u'UPDATE chipseq SET "PATH_JOB_FILE"=? WHERE chipseq."SAMPLE_ID" = ?'] [parameters: ('/users/GR/mb/jquilez/pipelines/chipseq-16.04/job_cmd/gv_020_01_01_chipseq_2016_04_27_full_macs2_chipseq-16.04.sh', 'gv_020_01_01_chipseq')]
INFO 	2016-04-27 15:36:08 	[trim_reads_trimmomatic] 	paths to read1 saved to metadata database
INFO 	2016-04-27 15:36:08 	[trim_reads_trimmomatic] 	sequencing type = SE
INFO 	2016-04-27 15:36:08 	[trim_reads_trimmomatic] 	trimming adapter sequences for HiSeq, NextSeq or HiSeq
INFO 	2016-04-27 15:36:08 	[trim_reads_trimmomatic] 	trimming low-quality reads ends using trimmomatic's recommended practices
INFO 	2016-04-27 16:03:23 	[trim_reads_trimmomatic] 	parse step log to extract generated metadata
INFO 	2016-04-27 16:03:23 	[trim_reads_trimmomatic] 	reads after trimming = 77270673
INFO 	2016-04-27 16:03:39 	[trim_reads_trimmomatic] 	trimmomatic parameters and numbe of trimmed reads added to metadata
INFO 	2016-04-27 16:03:39 	[trim_reads_trimmomatic] 	trimmed reads are in /users/GR/mb/jquilez/data/chipseq/samples/gv_020_01_01_chipseq/fastqs_processed/trimmomatic/single_end
TIME 	2016-04-27 16:03:39 	[trim_reads_trimmomatic] 	step time for completion (seconds) = 1687

INFO 	2016-04-27 16:03:41 	[align_bwa] 	align single-end reads with BWA
INFO 	2016-04-27 16:03:41 	[align_bwa] 	alignments are converted to BAM, sorted by genomic coordinates and multi-mappings filtered out
[main] Version: 0.7.12-r1039
[main] CMD: /software/mb/el6.3/bwa/bwa-0.7.12/bwa mem -t 8 -M -R @RG\tID:'gv_020_01_01_chipseq'\tLB:'gv_020_01_01_chipseq'\tPL:illumina\tPU:'gv_020_01_01_chipseq'\tSM:'gv_020_01_01_chipseq' -v 0 /users/GR/mb/jquilez/assemblies/homo_sapiens/hg38_mmtv/ucsc/hg38_mmtv_chr1-22XYMUn.fa /users/GR/mb/jquilez/data/chipseq/samples/gv_020_01_01_chipseq/fastqs_processed/trimmomatic/single_end/gv_020_01_01_chipseq_read1.fastq.gz
[main] Real time: 3425.837 sec; CPU: 26281.755 sec
[bam_sort_core] merging from 37 files...
INFO 	2016-04-27 17:23:38 	[align_bwa] 	reads aligned = 77270685
INFO 	2016-04-27 17:23:38 	[align_bwa] 	reads unique = 64671029
INFO 	2016-04-27 17:23:43 	[align_bwa] 	path to genome sequence FASTA and number of (uniquely) aligned reads added to metadata
TIME 	2016-04-27 17:23:56 	[align_bwa] 	step time for completion (seconds) = 4815

INFO 	2016-04-27 17:23:57 	[quality_alignments] 	general QC of the BAM (using qualimap's bamqc)
INFO 	2016-04-27 17:27:54 	[quality_alignments] 	parse step log to extract generated metadata
INFO 	2016-04-27 17:27:54 	[quality_alignments] 	percentage duplication = 10.91
INFO 	2016-04-27 17:27:54 	[quality_alignments] 	median insert size (bp) = 0
INFO 	2016-04-27 17:27:54 	[quality_alignments] 	mean mapping quality = 46.4772
INFO 	2016-04-27 17:27:54 	[quality_alignments] 	mean coverage (X) = 0.9597
INFO 	2016-04-27 17:27:54 	[quality_alignments] 	mean coverage adjusted paired-end = .
TIME 	2016-04-27 17:28:01 	[quality_alignments] 	step time for completion (seconds) = 244

INFO 	2016-04-27 17:28:02 	[make_tag_directory] 	converting BAM to BED --required for making tag directory
INFO 	2016-04-27 17:31:22 	[make_tag_directory] 	make tag directory with HOMER
INFO 	2016-04-27 17:42:00 	[make_tag_directory] 	parse step log to extract generated metadata
INFO 	2016-04-27 17:42:00 	[make_tag_directory] 	estimated genome size = 3090953370
INFO 	2016-04-27 17:42:00 	[make_tag_directory] 	estimated average read density per bp = 0.020923
INFO 	2016-04-27 17:42:00 	[make_tag_directory] 	total tags = 64671029.0
INFO 	2016-04-27 17:42:00 	[make_tag_directory] 	total_positions = 57839076
INFO 	2016-04-27 17:42:00 	[make_tag_directory] 	avg. tag length = 45.9
INFO 	2016-04-27 17:42:00 	[make_tag_directory] 	median tags per position = 1
INFO 	2016-04-27 17:42:00 	[make_tag_directory] 	avg. tags per position = 1.118
INFO 	2016-04-27 17:42:00 	[make_tag_directory] 	fragment length estimate = 124
INFO 	2016-04-27 17:42:00 	[make_tag_directory] 	peak width estimate = 194
INFO 	2016-04-27 17:42:00 	[make_tag_directory] 	autocorrelation: same strand fold enrichment = 1.2
INFO 	2016-04-27 17:42:00 	[make_tag_directory] 	autocorrelation: diff strand fold enrichment = 1.2
INFO 	2016-04-27 17:42:00 	[make_tag_directory] 	autocorrelation: same-to-diff strand fold enrichment = 1.0
INFO 	2016-04-27 17:42:00 	[make_tag_directory] 	avg. fragment GC% = 43.14
INFO 	2016-04-27 17:42:00 	[make_tag_directory] 	avg. expected GC% = 40.39
INFO 	2016-04-27 17:42:19 	[make_tag_directory] 	ChIP-seq metrics calculated with Homer added to metadata
TIME 	2016-04-27 17:42:19 	[make_tag_directory] 	step time for completion (seconds) = 857

INFO 	2016-04-27 17:42:20 	[make_profiles] 	generate reads per million profile (RPM) fragment profile
INFO 	2016-04-27 18:37:45 	[make_profiles] 	get fragment length estimate (l) as calculated in the 'make_tag_directory' step
INFO 	2016-04-27 18:37:45 	[make_profiles] 	fragment length estimate = 124
INFO 	2016-04-27 18:37:45 	[make_profiles] 	fragment length correction = 124
INFO 	2016-04-27 18:37:45 	[make_profiles] 	the correction will be used if the estimate is not reliable
TIME 	2016-04-27 18:37:47 	[make_profiles] 	step time for completion (seconds) = 3327

INFO 	2016-04-27 18:37:49 	[call_peaks] 	Get fragment length estimate (l) as calculated in the 'make_tag_directory' step
INFO 	2016-04-27 18:37:49 	[call_peaks] 	Fragment length (l) is 124 bp (note this is not used if peak caller is zerone)
INFO 	2016-04-27 18:37:49 	[call_peaks] 	Peak calling with MACS2
INFO 	2016-04-27 18:37:49 	[call_peaks] 	genome size for homo_sapiens will be used
INFO 	2016-04-27 18:37:49 	[call_peaks] 	q-value cutoff = 0.05 (default is 0.01)
INFO 	2016-04-27 18:37:49 	[call_peaks] 	--nomodel (MACS2 will not try to model peak length but use l = 124 instead
INFO 	2016-04-27 18:37:49 	[call_peaks] 	--call-summits = MACS reanalyzes the shape of signal profile to deconvolve subpeaks within each peak called
INFO 	2016-04-27 18:37:49 	[call_peaks] 	peak calling with the sample alone (i.e. no input)
INFO 	2016-04-27 18:46:00 	[call_peaks] 	peaks = 9926
INFO 	2016-04-27 18:46:06 	[call_peaks] 	peak calling without input DNA as control added to metadata

